Added support for Chex Quest 3 and Hacx v1.2

Shockwave508 opened the pull request:
Since these two games are forked off from Doom and Doom II, respectively, I decided to attempt an implementation of NCHUD support for Chex Quest 3 and HACX v1.2. Let me know how I did.

Shockwave508 committed:
Added Hacx 1.2 Support

Shockwave508 committed:
Added Hacx 1.2 Support

Shockwave508 committed:
Added Hacx 1.2 Support

Shockwave508 committed:
Added Chex Quest Support

Shockwave508 committed:
Added Chex Quest Support

Shockwave508 committed:
Create althudcf.txt

Shockwave508 committed:
Delete althudcf.txt

Shockwave508 committed:
Added Chex Quest Support

Shockwave508 committed:
Added Chex Quest Support

Blue-Shadow commented:
I'm okay with the feature itself, but I have to admit, I don't like how this spans multiple commits (of which there are 9!), considering how simple and small the additions are. You don't need to make a separate commit for each file you add, remove or edit. This pull request could be made in a single commit (or two: one for Chex and the other for Hacx), which is better.

Shockwave508 committed:
Added missing Radsuit graphic for Chex Quest

Shockwave508 committed:
Added missing Rubber Boots graphic for Hacx

Shockwave508 commented:
Still very much a rookie when it comes to GitHub, my apologies; keep in mind I'm doing this off of my Android phone, since I don't have access to a PC. ^^;

Anyway, I dug through the WAD files for Chex 3 and Hacx, and saw Radsuit graphics for both games, so I fixed it up a little and added them to the althudcf.txt files for both filters. As far as I know, there are no equivalents for the Invulnerability Sphere and BlurSphere in Chex Quest.
